bigtable/client/rpc_backoff_policy_test.cc

```diff
@@ -50,3 +50,20 @@ TEST(ExponentialBackoffRetryPolicy, Clone) {
   EXPECT_EQ(80_ms, tested->on_completion(CreateTransientError()));
   EXPECT_EQ(150_ms, tested->on_completion(CreateTransientError()));
 }
+
+TEST(RandomizedBackoffRetryPolicy, Simple) {
+  using namespace bigtable::chrono_literals;
+  bigtable::RandomizedBackoffPolicy test1(10_ms, 50_ms);
+
+  EXPECT_EQ(10_ms, test1.on_completion(CreateTransientError()));
+  EXPECT_LT(10_ms, test1.on_completion(CreateTransientError()));
+}
+
+TEST(RandomizedBackoffRetryPolicy, Clone) {
```

Comment:
Consider a test where you create two `RandomizedBackoffPolicy` objects and create a long (~100) series of numbers.  We would expect the series to be different, if all the numbers are identical we have a problem.

Reply:
Done.